We Got Coders - My Amazing Application.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
```

Comment:
What was the intention with saving and committing this particular file? It seems like this was captured from running your submission and saving the result which wasn't strictly necessary, in my reviewing of your submission this is something I would do normally anyway :+1: